crates/pop-contracts/src/node.rs

```diff
@@ -232,8 +232,8 @@ fn release_directory_by_target(binary: &str) -> Result<Vec<ArchiveFileSpec>, Err
 mod tests {
 	use super::*;
 	use anyhow::{Error, Result};
-	use pop_common::find_free_port;
-	use std::process::Command;
+
+	const POLKADOT_NETWORK_URL: &str = "wss://polkadot-rpc.publicnode.com";
```

Comment:
Wanted to ask here if it wouldn't be better bringing the endpoint from the constants added in this same PR.
Then I realized that by doing so we would be making `pop-contracts` depend on `pop-cli`. Which is not what we want.

This makes me think if some of the changes introduced into `pop-cli` for testing might be better on `pop-common` 🤔 I don't think it's worth blocking this PR at this point for this.